src/components/four-up.test.tsx

```diff
@@ -25,6 +25,11 @@ jest.mock("../hooks/use-stores", () => ({
   }),
   useUIStore: () => ({
     showAdornments: true
+    // TODO: FIXME - doesn't seem like showAdornments is being used (since missing from UIModel)
+    // but useUIStore needs to be defined
+  }),
+  usePersistentUIStore: () => ({
+    showAnnotations: true
```

Comment:
I'd guess this isn't really needed? So you could try removing it and see if the test still passes.

Reply:
see comment above...